### PR TITLE
tables / date picker

### PR DESCRIPTION
**Pull request by eloyEntony**
update tables / date picker

## Summary by CodeRabbit

- **New Features**
	- Enhanced table component with improved cell alignment, date comparison logic, and badge rendering based on date values.
- **Refactor**
	- Updated sorting functionality in tables to support a wider range of data types.
	- Optimized import paths for better module management.
- **Style**
	- Adjusted global text line-height for improved readability and layout consistency.

### CODE REVIEW

**Comment by rhahao** [comment]
Task linked: [CU-86bxxpwnb Tables](https://app.clickup.com/t/86bxxpwnb)

File: src/v3/hooks/useSorting.ts
Comment (rhahao) [inline src/v3/hooks/useSorting.ts:1]: @eloyEntony: would it be possible to move this file according to its current location in the `main-next` branch. We will only use this folder for global hooks, but this seems to be related to the table component only.

File: src/v3/global.css [hunk 1]
```diff
@@ -598,7 +598,7 @@
   font-style: normal;
   font-stretch: normal;
   letter-spacing: -0.12px;
-  line-height: 16px;
+  line-height: 14px;
```
Comment (rhahao) [inline src/v3/global.css:601]: @eloyEntony: have already have confirmation about this change from the designers?

**Review by rhahao — CHANGES_REQUESTED** [review]

File: src/v3/global.css (hunk 1 above)
Comment (ux-git) [reply src/v3/global.css:601]: @rhahao yes, it has been discussed. Hopefully this change won't cause more further issues on next stages 😀

**Review by rhahao — APPROVED** [review]

**Pull request merged**